src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/InterceptorConfig.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.overseasentitiesapi.configuration;
+
+import uk.gov.companieshouse.overseasentitiesapi.Interceptor.LoggingInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@ComponentScan("uk.gov.companieshouse.api.interceptor")
```

Comment:
the component scan is not needed yet, it is scanning package uk.gov.companieshouse.api.interceptor which we don't have